src/components/App/App.js

```diff
@@ -25,140 +20,138 @@ import EditUser from '../EditUser/EditUser';
-            <ProtectedRoute
-              // logged in shows UserPage else shows LoginPage
-              exact
-              path="/user"
-              component={UserPage}
-            />
-            <ProtectedRoute
-              // logged in shows UserPage else shows LoginPage
-              exact
-              path="/userhub"
-              component={UserHub}
-            />
-            <ProtectedRoute
-              // logged in shows UserPage else shows LoginPage
-              exact
-              path="/edituser"
-              component={EditUser}
-            />
-            <ProtectedRoute
-              // logged in shows UserPage else shows LoginPage
-              exact
-              path="/editdeck/:id"
-              component={EditDeck}
-            />
-             <ProtectedRoute
-              // logged in shows UserPage else shows LoginPage
-              exact
-              path="/asyncinput"
-              component={Asyncinput}
-            />
-            <ProtectedRoute
-              // logged in shows UserPage else shows ViewDeck
-              exact
-              path="/viewdeck/:id"
-              component={ViewDeck}
-            />
-
-            
-            <ProtectedRoute
-              // logged in shows InfoPage else shows LoginPage
-              exact
-              path="/info"
-              component={InfoPage}
-            />
+        <Nav />
+        <Switch>
+          <Redirect exact from="/" to="/home" />
+          
+          {/* UNPROTECTED ROUTES */}
+          <Route
+            exact
+            path="/about"
+            component={AboutPage}
+          />
+          <Route
+            exact
+            path="/lifetotal"
+            component={LifeTotal}
+          />
+          <Route
+            // no longer using this
+            exact
+            path="/asyncinput"
+            component={Asyncinput}
+          />
+          <Route
+            // logged in shows ViewDeck else shows ViewDeck
+            exact
+            path="/viewdeck/:id"
+            component={ViewDeck}
+          />
+          <Route
+            // logged in shows InfoPage else shows LoginPage
+            exact
+            path="/info"
+            component={InfoPage}
+          />  
+          {/* For protected routes, the view could show one of several things on the same route.
+          Visiting localhost:3000/user will show the UserPage if the user is logged in.
+          If the user is not logged in, the ProtectedRoute will show the LoginPage (component).
+          Even though it seems like they are different pages, the user is always on localhost:3000/user */}
+          
+          {/* NO LONGER USING THIS /user ROUTE */}
+          {/* <ProtectedRoute
+            // logged in shows UserPage else shows LoginPage 
+            exact
+            path="/user"
+            component={UserPage}
+          /> */}
```

Comment:
Do you still need/want the protected route comment?

If the userpage route isn't used can it be deleted?